docs/securitypolicy.md

```diff
@@ -61,6 +63,6 @@ Everything from this point and below should be in the `Security Options` pane of
 -   `NewAdministratorName`
 -   `NewGuestName`
 
-## Security Options
+## Other Options
 
-I'm seriously going to suffer brain damage if I have to format all of these again... [See the spreadsheet](https://docs.google.com/spreadsheets/d/1N7uuke4Jg1R9FBhj8o5dxJQtEntQlea0McYz5upaiTk/edit#gid=1772229936).
```

Comment:
Can you leave the spreadsheet? It was kinda helpful
also 😭😭 not the funny messages

Reply:
I'm planning on deleting the account that's hosting that, and I think it was a bit misleading since it had values that weren't implemented in the engine. But if someone else wanted to host it in, then we could add it back haha.